blackbox/docs/general/builtins/aggregation.rst

```diff
@@ -13,33 +13,91 @@ Aggregation
 Introduction
 ============
 
-An *aggregation function* operates on multiple values of the specified column
-of your ``SELECT``. For example, ``SELECT avg(temperature)`` will return the
-average value of the ``temperature`` column across all rows being considered.
+An **aggregate function** computes a single result from a set of input values.
 
-You can apply aggregation functions to the results of a whole query, or to each
-of the rows produced by a :ref:`sql_dql_group_by` query.
+::
```

Comment:
Reduced a bit amount of text and add simple examples instead of their textual description.